packages/grafana-ui/src/components/Icon/Icon.tsx

```diff
@@ -19,6 +19,15 @@ export interface IconProps extends Omit<React.SVGProps<SVGElement>, 'onLoad' | '
   title?: string;
 }
 
+const spin = keyframes({
+  '0%': {
+    transform: 'rotate(0deg)',
+  },
+  '100%': {
+    transform: 'rotate(359deg)',
+  },
+});
+
```

Comment:
Could we extract this keyframe as it's used in a few places? Also thinking how to add this to the icons package 🤔

Reply:
yeah, but i don't really know where to put it 😂 any suggestions? utils folder? 🤔 

imo we shouldn't put it in the icon package. i think it's a mistake that it's here in the first place, but we kinda have to keep it here for backwards compatibility. think consumers should be responsible for animating icons, and the icon package should just provide the icon. wdyt?

Reply:
> yeah, but i don't really know where to put it 😂 

I'd just put it into `utils/keyframes.ts`

> think consumers should be responsible for animating icons, and the icon package should just provide the icon. wdyt?

Yeah, sounds good, also easier for us 😄